build/mark-new-version.sh

```diff
@@ -93,15 +94,13 @@ echo "+++ Running ./versionize-docs"
 ${KUBE_ROOT}/build/versionize-docs.sh ${NEW_VERSION}
 git commit -am "Versioning docs and examples for ${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}"
 
-dochash=$(git log -n1 --format=%H)
-
 VERSION_FILE="${KUBE_ROOT}/pkg/version/base.go"
 
 GIT_MINOR="${VERSION_MINOR}.${VERSION_PATCH}"
 echo "+++ Updating to ${NEW_VERSION}"
 $SED -ri -e "s/gitMajor\s+string = \"[^\"]*\"/gitMajor string = \"${VERSION_MAJOR}\"/" "${VERSION_FILE}"
 $SED -ri -e "s/gitMinor\s+string = \"[^\"]*\"/gitMinor string = \"${GIT_MINOR}\"/" "${VERSION_FILE}"
```

Comment:
Honestly, I was half tempted to follow the `TODO` in `base.go` and kill the gitMajor/gitMinor fields, but I was afraid of scripts consuming them. This almost lends credence to no one consuming them usefully, because the "official" builds don't actually have anything useful there.